Add blog post for sprint 106 SCM/CI integration (#323)
openSUSE/obs-landing: _includes/partials/_series-of-posts-about-scm-integration.md

We introduced advanced features like [reporting filters and support for self-hosted SCM](/2021/07/12/scm-integration-more-improvements)
together with a list of [common pitfalls](/2021/07/12/scm-integration-pitfalls) in July 2021
and in August 2021, we continued with [two new steps and a UI for tokens](/2021/08/02/link-package-configure-repositories-and-ui).

> We started to [support more actions for pull/merge requests and improving the UI for tokens](/2021/09/14/scm-more-pull-request-actions-and-ui),

Reviewer: ```suggestion
After [supporting more actions for pull/merge requests and improving the UI for tokens](/2021/09/14/scm-more-pull-request-actions-and-ui),
```

and [adding support for push events and a rebuild step](/2021/09/28/support-for-push-events) in September 2021.

> An now we present [user documentation and further improvements for the UI for tokens](/2021-11-09-scm-integration-documentation)._

Reviewer: ```suggestion
We now present [user documentation and further improvements for the UI for tokens](/2021-11-09-scm-integration-documentation)._
```
